torchvision/datasets/video_utils.py

```diff
@@ -198,29 +200,32 @@ def subset(self, indices):
         )
 
     @staticmethod
-    def compute_clips_for_video(video_pts, num_frames, step, fps, frame_rate):
+    def compute_clips_for_video(
+        video_pts: torch.Tensor, num_frames: int, step: int, fps: int, frame_rate: Optional[int] = None
+    ) -> Tuple[torch.Tensor, Union[List[slice], torch.Tensor]]:
         if fps is None:
             # if for some reason the video doesn't have fps (because doesn't have a video stream)
             # set the fps to 1. The value doesn't matter, because video_pts is empty anyway
             fps = 1
         if frame_rate is None:
             frame_rate = fps
         total_frames = len(video_pts) * (float(frame_rate) / fps)
-        idxs = VideoClips._resample_video_idx(int(math.floor(total_frames)), fps, frame_rate)
```

Comment:
What's the point of this change in variable name?

Reply:
We are overwriting it with a different type below and `mypy` doesn't like that. See https://github.com/pytorch/vision/pull/4172/files/b8887ec9449ba5e8719ab9cf1c44c375d9ecb22e#r752061997